lib/components/overlays/launcher/widgets/app_launcher_button.dart

```diff
@@ -54,7 +55,9 @@ class _AppLauncherButtonState extends State<AppLauncherButton> {
               focusColor: CommonData.of(context).textColor(),
               onTap: () {
                 if (application.systemExecutable == true) {
-                  print(application.runtimeFlags.toString());
+                  if (kDebugMode) {
+                    print(application.runtimeFlags.toString());
+                  }
```

Comment:
a special log function that handles this automatically would be better

Reply:
where will be a good place for such function?